gnomad_qc/v4/resources/annotations.py

```diff
@@ -14,19 +14,22 @@
 SUBSETS = SUBSETS["v4"]
 
 
-def _annotations_root(version: str = CURRENT_VERSION, test: bool = False) -> str:
+def _annotations_root(
+    version: str = CURRENT_VERSION, data_type: str = "exomes", test: bool = False
```

Comment:
I suggest we move data_type after test incase anything just uses parameter ordering
```suggestion
    version: str = CURRENT_VERSION, test: bool = False,  data_type: str = "exomes",
```